package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ungit",
   "author": "Fredrik Norén <fredrik.jw.noren@gmail.com>",
   "description": "Git made easy",
-  "version": "1.4.6",
+  "version": "1.4.7",
```

Comment:
also update `package-lock.json`

Reply:
I don't think they changes `package-lock.json` for version up for the actual app.  I have no changes for it.

Reply:
Hmm if I run `npm install` it shows up as a change. But it might not be important.

![image](https://user-images.githubusercontent.com/4009570/35185559-d8fdcda4-fe06-11e7-9f1d-dc1c67f6dec3.png)

Reply:
you are absolutely right and it is now fixed.  will do `npm i` onwards even if no dep changes happens.